1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,32 @@
 function readingList(books) {
-  // Write your code here...
+  let ulElement = document.createElement("ul")
+  content.appendChild(ulElement)
+  for (let book of books) { 
+    let pElement = document.createElement("p")
+    
+    pElement.innerText = `${book.title} by ${book.author} `
+
+    let imgElement = document.createElement("img")
+    imgElement.setAttribute("src", book.bookCoverImage)
```

Comment:
Nice way of adding image by setAttribute, fyi here is another way that also works `imgElement.src`.